lib/core/logconf.c

```diff
@@ -831,18 +847,28 @@ char *h2o_log_request(h2o_logconf_t *logconf, h2o_req_t *req, size_t *len, char
             RESERVE(sizeof(H2O_UINT64_LONGEST_STR) - 1);
             pos += sprintf(pos, "%" PRIu64, req->proxy_stats.bytes_read.body);
             break;
-
+        case ELEMENT_TYPE_PROXY_SSL_SESSION_REUSED:
+            RESERVE(1);
+            *pos++ = (req->proxy_stats.ssl.session_reused) ? '1' : '0';
+            break;
+        case ELEMENT_TYPE_PROXY_SSL_CIPHER_BITS:
+            if (req->proxy_stats.ssl.cipher_bits == 0)
+                goto EmitNull;
+            RESERVE(sizeof(H2O_INT16_LONGEST_STR));
+            pos += sprintf(pos, "%" PRIu16, req->proxy_stats.ssl.cipher_bits);
```

Comment:
While it is fine to assume that the number of bits is below UINT16_MAX, I think we need to cast `req->proxy_stats.ssl.cipher_bits` to `uint16_t` explicitly. The type of the variable is `int`.

Reply:
oh. shame on me. fixed